float64[7] dq_error
float64[7] dq_commanded
float64[7] dq_measured
float64 root_mean_square_error
